CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Ember Flexberry Changelog
 ### New Features & improvements
 
+## 2016-05-24
+### Breaking changes
+Addon change:
+* Delete dependency from ember-simple-auth.
+* Delete dependency from ember-local-storage.
+
```

Comment:
I think it would be useful to add here changes about remove services, authorizer, authenticator, controller and so on, i.e. information about removed parts of addon.